.phpcs/autoload.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+// Since doctrine/coding-standard requires PHP 7, we can't add it as a dependency
+// yet. This autoload file adds more information to the phpcs error message,
+// telling the user how they can fix the error presented to them by phpcs.
+if (! file_exists(__DIR__ . '/../vendor/doctrine/coding-standard')) {
```

Comment:
Since doctrine/coding-standard requires PHP 7.2, including it in `require-dev` would implicitly bump our PHP requirement to 7.2 on CI systems, breaking tests for older PHP versions. That's why the check stage requires it manually.

To make the generic error message from phpcs more helpful I decided to use the autoload functionality of phpcs to include this error message if doctrine/coding-standard wasn't installed. Now that you mention it, it's probably best to also mention this in the contribution docs.